createCard/index.spec.js

```diff
@@ -24,7 +26,9 @@ describe( "createCard", () => {
 			boardId: "BOARDID",
 			title: "TITLE",
 			laneId: "LANEID",
-			typeId: "TYPEID"
+			typeId: "TYPEID",
+            customId: "CUSTOMID",
+            externalLink: "EXTERNALLINK"
```

Comment:
This should be `externalLink: { label: "Link to Github", url: "EXTERNALLINK" }`, although this might change if you start collecting a label.

Reply:
I'll put it as per your suggestion right now and then change it when I get to do the next bit